zproject/prod_settings_template.py

```diff
@@ -366,6 +366,10 @@
         ## reads the secret with the specified name from zulip-secrets.conf.
         "client_id": "<your client id>",
         "secret": get_secret("social_auth_oidc_secret"),
+        ## Equivalent of auto_signup in SAML configuration. If set to True,
+        ## the page asking the user to confirm they intend to register a new
+        ## account will be skipped when they authenticate through the "Login" page.
```

Comment:
I copied the original SAML documentation here, since this is user-facing documentation and thus we'd like it to be self-contained.